### ISSUE TITLE
pretrained_model 的bert_en 内容不全吗

### ISSUE THREAD

**Issue by zhaoyuanyi** [thuiar/Self-MM#32]
Model name 'pretrained_model/bert_en' not found in model shortcut name list (bert-base-uncased, bert-large-uncased, bert-base-cased, bert-large-cased, bert-base-multilingual-uncased, bert-base-multilingual-cased, bert-base-chinese, bert-base-german-cased, bert-large-uncased-whole-word-masking, bert-large-cased-whole-word-masking, bert-large-uncased-whole-word-masking-finetuned-squad, bert-large-cased-whole-word-masking-finetuned-squad, bert-base-cased-finetuned-mrpc, bert-base-german-dbmdz-cased, bert-base-german-dbmdz-uncased, bert-base-finnish-cased-v1, bert-base-finnish-uncased-v1, bert-base-dutch-cased). Assuming 'pretrained_model/bert_en' is a path, a model identifier, or url to a directory containing tokenizer files.
Didn't find file pretrained_model/bert_en/added_tokens.json. We won't load it.
Didn't find file pretrained_model/bert_en/special_tokens_map.json. We won't load it.
Didn't find file pretrained_model/bert_en/tokenizer_config.json. We won't load it.
loading file pretrained_model/bert_en/vocab.txt
loading file None
loading file None
loading file None
loading configuration file pretrained_model/bert_en/config.json
Model config BertConfig {
  "attention_probs_dropout_prob": 0.1,
  "hidden_act": "gelu",
  "hidden_dropout_prob": 0.1,
  "hidden_size": 768,
  "initializer_range": 0.02,
  "intermediate_size": 3072,
  "layer_norm_eps": 1e-12,
  "max_position_embeddings": 512,
  "model_type": "bert",
  "num_attention_heads": 12,
  "num_hidden_layers": 12,
  "pad_token_id": 0,
  "type_vocab_size": 2,
  "vocab_size": 30522

**Comment by soyorin33** [comment]
同问，请问解决了吗
